torch/onnx/__init__.py

```diff
@@ -119,9 +119,7 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
             output nodes of the graph, in order.
         operator_export_type (enum, default None):
 
-            None usually means ``OperatorExportTypes.ONNX``.
-            However if PyTorch was built with ``-DPYTORCH_ONNX_CAFFE2_BUNDLE``, None means
-            ``OperatorExportTypes.ONNX_ATEN_FALLBACK``.
+            None defaults to ``OperatorExportTypes.ONNX``.
```

Comment:
change the actual default from `None` to `OperatorExportTypes.ONNX` and remove the logic in the code that checks for None.